javascript/test/ParserTest.ts

```diff
@@ -308,5 +309,128 @@ description
+      const markdown = `## Feature: DataTables
+
+### Scenario: minimalistic
+
+* Given a simple data table 
+  | foo | bar |
+  | --- | --- |
+  | boz | boo |
+`
+      const ast = parser.parse(markdown)
+      const gherkinDocument: messages.GherkinDocument = {
+        "comments": [
+          {
+            "location": {
+              "column": 3,
+              "line": 7
+            },
+            "text": undefined
+          }
+        ],
+        "feature": {
+          "children": [
+            {
+              "scenario": {
+                "description": "",
+                "examples": [],
+                "id": "3",
+                "keyword": "Scenario",
+                "location": {
+                  "column": 5,
+                  "line": 3
+                },
+                "name": "minimalistic",
+                "steps": [
+                  {
+                    "dataTable": {
+                      "location": {
+                        "column": 3,
+                        "line": 6
+                      },
+                      "rows": [
+                        {
+                          "cells": [
+                            {
+                              "location": {
+                                "column": 5,
+                                "line": 6
+                              },
+                              "value": "foo"
+                            },
+                            {
+                              "location": {
+                                "column": 11,
+                                "line": 6
+                              },
+                              "value": "bar"
+                            }
+                          ],
+                          "id": "0",
+                          "location": {
+                            "column": 3,
+                            "line": 6
+                          }
+                        },
+                        {
+                          "cells": [
+                            {
+                              "location": {
+                                "column": 5,
+                                "line": 8
+                              },
+                              "value": "boz"
+                            },
+                            {
+                              "location": {
+                                "column": 11,
+                                "line": 8
+                              },
+                              "value": "boo"
+                            }
+                          ],
+                          "id": "1",
+                          "location": {
+                            "column": 3,
+                            "line": 8
+                          }
+                        }
+                      ]
+                    },
+                    "id": "2",
+                    docString: undefined,
```

Comment:
this attribute is required in the Javascript implementation, but not in the Python implementation.